firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -4127,7 +4127,7 @@ extension String {
                 comment: "Text to let users know how many trackers were blocked on the current website. Placeholder for the number of trackers blocked")
 
             public static let noTrackersLabel = MZLocalizedString(
-                key: "Menu.EnhancedTrackingProtection.Details.Trackers.v128",
+                key: "Menu.EnhancedTrackingProtection.Details.NoTrackers.v131",
```

Comment:
As per our [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#how-to-modify-a-string), lets add the old key to our old struct.

Reply:
Discussed with @adudenamedruby and @dicarobinho, will keep as is and change if l10n requested